nesta/core/batchables/nlp/bert_vectorize/run.py

```diff
@@ -65,15 +65,29 @@ def run():
     # Convert text to vectors
     embeddings = model.encode(docs)
     # Write to output
-    out_data = [{"article_id": _id, "vector": embedding.tolist()}
```

Comment:
this was a bug from the `arxiv` vectorisation pipeline (which was the only case that `bert_vectorize` was used in) where `id_field="article_id"`, so just replaced with the generic variable `id_field` as required